addons/account/models/chart_template.py

```diff
@@ -168,6 +168,10 @@ def _load(self, template_code, company, install_demo):
         )
         company = company.with_env(self.env)
 
+        old_accounts = self.env['account.account'].search([
```

Comment:
Renamed to `preexisting_accounts`. With `existing_account` it seemed a bit weird in the computation of  `new_accounts_without_xmlid` since all accounts should exist at that point.